<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <author>Mohamed Salah</author>
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="enabled">
            <bool>true</bool>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>323</width>
                <height>561</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>InstaHelper</string>
        </property>
        <property name="windowIcon">
            <iconset>
                <normaloff>../../../../.designer/backup/src/InstaHelper.png</normaloff>
                ../../../../.designer/backup/src/InstaHelper.png
            </iconset>
        </property>
        <property name="windowOpacity">
            <double>1.000000000000000</double>
        </property>
        <widget class="QWidget" name="centralWidget">
            <property name="enabled">
                <bool>true</bool>
            </property>
            <widget class="QLabel" name="appName">
                <property name="geometry">
                    <rect>
                        <x>60</x>
                        <y>0</y>
                        <width>191</width>
                        <height>91</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Ubuntu</family>
                        <pointsize>25</pointsize>
                        <weight>75</weight>
                        <italic>false</italic>
                        <bold>true</bold>
                    </font>
                </property>
                <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                    <string>InstaHelper</string>
                </property>
                <property name="scaledContents">
                    <bool>false</bool>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                    <bool>false</bool>
                </property>
                <property name="buddy">
                    <cstring>appName</cstring>
                </property>
            </widget>
            <widget class="QLabel" name="status">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>460</y>
                        <width>291</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Enter Instagram username and password</string>
                </property>
            </widget>
            <widget class="QGroupBox" name="loginBox">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>80</y>
                        <width>281</width>
                        <height>371</height>
                    </rect>
                </property>
                <property name="title">
                    <string>Login</string>
                </property>
                <widget class="QLineEdit" name="usernameEdit">
                    <property name="enabled">
                        <bool>true</bool>
                    </property>
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>80</y>
                            <width>261</width>
                            <height>51</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>20</pointsize>
                        </font>
                    </property>
                    <property name="autoFillBackground">
                        <bool>false</bool>
                    </property>
                    <property name="inputMask">
                        <string/>
                    </property>
                    <property name="text">
                        <string/>
                    </property>
                    <property name="placeholderText">
                        <string>Type your username</string>
                    </property>
                </widget>
                <widget class="QLineEdit" name="passwordEdit">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>150</y>
                            <width>261</width>
                            <height>51</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>20</pointsize>
                        </font>
                    </property>
                    <property name="inputMethodHints">
                        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
                    </property>
                    <property name="echoMode">
                        <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="placeholderText">
                        <string>Type your password</string>
                    </property>
                </widget>
                <widget class="QCheckBox" name="rememberCheckBox">
                    <property name="geometry">
                        <rect>
                            <x>20</x>
                            <y>240</y>
                            <width>241</width>
                            <height>41</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Remmber me</string>
                    </property>
                    <property name="checked">
                        <bool>true</bool>
                    </property>
                </widget>
                <widget class="QPushButton" name="loginButton">
                    <property name="geometry">
                        <rect>
                            <x>80</x>
                            <y>300</y>
                            <width>121</width>
                            <height>51</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Login</string>
                    </property>
                    <property name="shortcut">
                        <string>Return</string>
                    </property>
                    <property name="flat">
                        <bool>false</bool>
                    </property>
                </widget>
            </widget>
            <zorder>loginBox</zorder>
            <zorder>appName</zorder>
            <zorder>status</zorder>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
    </widget>
    <tabstops>
        <tabstop>usernameEdit</tabstop>
        <tabstop>passwordEdit</tabstop>
        <tabstop>rememberCheckBox</tabstop>
        <tabstop>loginButton</tabstop>
    </tabstops>
    <resources/>
    <connections/>
</ui>
